zilencer/management/commands/populate_db.py

```diff
@@ -204,9 +205,13 @@ def handle(self, **options):
             zulip_realm_bots = [
                 ("Zulip New User Bot", "new-user-bot@zulip.com"),
                 ("Zulip Error Bot", "error-bot@zulip.com"),
+                ("Zulip Webhook Bot", "webhook-bot@zulip.com"),
                 ]
+
             zulip_realm_bots.extend(all_realm_bots)
             create_users(realms, zulip_realm_bots, bot=True)
+            webhook_bot = UserProfile.objects.get(email="webhook-bot@zulip.com")
+            do_change_is_incoming_webhook(webhook_bot, True)
```

Comment:
Though probably eventually we'll want to refactor this to make the variable we pass be `bot_type` (where null means not a bot) rather than a `bot` bool so we can skip the `do_change_is_incoming_webhook` line.